index1.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <p id="demo"></p>
```

Comment:
same thing should want to do. this one also should want to do in the index.html file. use demo1 p element for the append this output.

Reply:
@nethmiprathibha this file also should want to delete.